contribs/src/main/java/com/netflix/conductor/contribs/kafka/KafkaProducerManager.java

```diff
@@ -0,0 +1,50 @@
+package com.netflix.conductor.contribs.kafka;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.netflix.conductor.core.config.Configuration;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.LongSerializer;
+
+import java.util.Objects;
+import java.util.Properties;
+
+public class KafkaProducerManager {
+
+	public static final String KAFKA_PUBLISH_REQUEST_TIMEOUT_MS = "kafka.publish.request.timeout.ms";
+	public static final String STRING_SERIALIZER = "org.apache.kafka.common.serialization.StringSerializer";
+	public String defaultRequestTimeOut = "100";
```

Comment:
This look constant. Have we documented it anywhere?

Reply:
Also, make this final and get default value from `configuration.getProperty` below.